double_helix/z_mapping.py

```diff
@@ -171,10 +177,25 @@ def lookup_dh_z(fres, calibration, rough_knot_spacing=101., plot=False):
         lobesep_residual = lobesep - lobesep_spline(z_out)
         sigma_residual = sigma - sig_spline(z_out)  # FIXME - should we be taking absolute value on sigma?
 
+        # compute the x-y distance between initial guess and fitted x-y coordinate
+        xy_initial_residual = np.empty_like(theta)
+        xy_initial_residual = np.sqrt((x - startParams_x0)**2 + (y - startParams_y0)**2)
+
+        # for each localization, compute and save the ratio of lobe amplitudes
+        # currently saved as ratio of larger amplitude to smaller amplitude (values 1<=)
+        A_ratio = np.empty_like(theta)
+        for ind in range(len(A0)):
+            if A0[ind] > A1[ind]:
+                A_ratio[ind] =  A0[ind]/A1[ind]
+            else:
+                A_ratio[ind] =  A1[ind]/A0[ind]
+

```

Comment:
should be performant to do something like
amplitudes = np.stack([chan['fitResults_A0'], chan['fitResults_A1']], axis=1)
amplitudes = np.sort(amplitudes, axis=1)
chan.addColumn('A_ratio', amplitudes[:,0] / amplitudes[:,1])